index.bs

```diff
@@ -3488,6 +3505,49 @@ While the best path forward may be to choose not to specify the feature,
 there is the risk that some implementations
 may ship the feature as a nonstandard API.
 
+<h3 id="multiple-algorithms">Write only one algorithm to accomplish a goal</h3>
+
+When specifying how to accomplish a goal, write a single algorithm to do it,
+instead of letting implementers pick between multiple algorithms.
+
+It is very difficult to ensure that
+two different algorithms produce the same results in all cases,
+and doing so is rarely worth the cost.
```

Comment:
```suggestion
<h3 id="single-algorithms">Write only one algorithm to accomplish a goal</h3>

When defining how to achieve a feature, it's better to specify a single
approach rather than offering multiple options.

If multiple methods are allowed, they must be equivalent in conformance to
avoid unnecessary complexity, inconsistency, and security risks.
```